presto-main/src/main/java/com/facebook/presto/metadata/FunctionManager.java

```diff
@@ -116,6 +106,16 @@ public FunctionHandle resolveOperator(OperatorType operatorType, List<TypeSignat
         return staticFunctionNamespace.resolveOperator(operatorType, argumentTypes);
     }
 
+    /**
+     * Lookup up a function with a fully qualified name and fully bound types.
```

Comment:
@rongrong Is this comment correct. Does the string `name` represent `fully qualified name`? The commit message suggests otherwise: `It should only work for internal functions. There's no need to use a QualifiedName. Other function
namespaces should not have this interface.`

Reply:
I updated the comment.